execution.bs

```diff
@@ -3710,6 +3718,13 @@ namespace std::execution {
   using <i>exec-envs</i>::<i>empty-env</i>;
   using <i>exec-envs</i>::get_env_t;
   using <i>exec-envs</i>::forwarding_env_query_t;
+
+  template&lt;class T>
+    concept environment =
```

Comment:
_Enviroment query_ eq is an CPO, such that for the expression e, such that `delctype(e)` models ` environment`, and some pack of expressions `args.`. the `eq(e, args...)` produces values of the corresponding property if it is well-formed.

For the expression `e`, `decltpe(e)` models the `environment` if for all environment equery `eq` and pack of expressions `args`, if `requires { eq(e, args...) }` is `true` then  `eq(e, args...)` is well-formed

Reply:
Maybe instead of 
> then `envquery(e, args...)` is well-formed

We could say, that would be cleaner that just saying is well-formed
> then  `envquery(e, args...)` meets the semantic requirement imposed by `envquery`.

Reply:
And for the introduction of enviroment equery, I think that something based on your existing wording would be better, like:
An _environment query_ is a customization point object that accepts as its first argument an environment, and for each such invocation that is valid, produces a value of the corresponding property of the environment.